[gamemodes]/[play]/play/play_players.lua

```diff
@@ -0,0 +1,35 @@
+local spawnsCount = #playerSpawns
+local skinsCount = #playerSkins
+
+function playSpawnPlayer(playerElement)
+	if isElement(playerElement) then
+		local randomSpawn = math.random(spawnsCount)
+		local spawnData = playerSpawns[randomSpawn]
+		local posX, posY, posZ, rotX = spawnData[1], spawnData[2], spawnData[3], spawnData[4]
```

Comment:
Sorry I missed this one. Let's use [unpack](https://www.lua.org/manual/5.1/manual.html#pdf-unpack) here as well.
```suggestion
		local posX, posY, posZ, rotX = unpack(spawnData)
```